sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/FieldShouldNotBePublic.cs

```diff
@@ -29,7 +29,7 @@ private class MyPrivateSubClass
 
         protected class MyProtectedSubClass
         {
-            public double Pi4 = 3.14;
+            public double Pi4 = 3.14;  // Noncompliant

```

Comment:
Double-check (if you haven't already) that the behaviour is consistent with rspec.